doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,302 @@
+* If a debugger (for example, J-Link) is connected via SWD to the nRF9160, the modem firmware will reset.
+  Therefore, the LTE modem cannot be operational during debug sessions.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+
+Crypto
+======
+
+* The :ref:`nrfxlib:nrf_security` glue layer is broken because symbol renaming is not handled correctly.
+  Therefore, the behavior is undefined when selecting multiple back-ends for the same algorithm (for example, AES).
+
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* :ref:`peripheral_lbs` does not report the Button 1 state correctly.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+* The :ref:`gatt_dm_readme` is not supported on nRF51 devices.
+* Bluetooth LE samples cannot be built with the :ref:`nrfxlib:ble_controller` v0.1.0.
+
+Bootloader
+----------
+
+* Building and programming the immutable bootloader (see :ref:`ug_bootloader`) is not supported in SEGGER Embedded Studio.
+* The immutable bootlader can only be used with the following boards:
+
+  * nrf52840_pca10056
+  * nrf9160_pca10090
+
+DFU
+---
+
+* Firmware upgrade using mcumgr or USB DFU is broken for multi-image builds, because the device tree configuration is not used.
+  Therefore, it is not possible to upload the image.
+  To work around this problem, build MCUboot and the application separately.
+
+
+nrfxlib
+=======
+
+* GNSS sockets implementation is experimental.
+
+ * Implementation might hard-fault when GPS is in running mode and messages are not read fast enough.
+ * NMEA strings are valid c-strings (0-terminated), but the read function might return wrong length.
+ * Sockets can only be closed when GPS is in stopped mode.
+ * Closing a socket does not properly clean up all memory resources.
+   If a socket is opened and closed multiple times, this  might starve the system.
+ * Forcing a cold start and writing AGPS data is not yet supported.
+
+nrfx 1.7.1
+==========
+
+* nrfx_saadc driver:
+  Samples might be swapped when buffer is set after starting the sample process, when more than one channel is sampled.
+  This can happen when the sample task is connected using PPI and setting buffers and sampling are not synchronized.
+* The nrfx_uarte driver does not disable RX and TX in uninit, which can cause higher power consumption.
+* The nrfx_uart driver might incorrectly set the internal tx_buffer_length variable when high optimization level is set during compilation.
```

Comment:
I'd change the last part of the sentence to "when compiled with a high optimization level" (as currently it sounds a bit like the optimization level could be changed while the compilation is in progress).
And perhaps "may" would be better than "might" (same in the nrfx_saadc driver entry).

Reply:
Changed the part of the sentence.  'Might' is preferred over 'may' as per our styleguide. Thanks for the review!